src/profile-logic/import/chrome.js

```diff
@@ -412,6 +422,155 @@ function makeFunctionInfoFinder(categories) {
   };
 }
 
+function processCpuProfile(
+  cpuProfile,
```

Comment:
Please provide explicit type information for this function. It's probably inferred, but would be good to have explicit types for it.